xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_support.xsd

```diff
@@ -497,4 +497,219 @@ Rail transport, Roads and Road transport
+			<xsd:enumeration value="topAndBottom"/>
+		</xsd:restriction>
+	</xsd:simpleType>
+	<!-- ======================================================================= -->
+	<xsd:simpleType name="StepConditionEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values forthe Step Condition</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="even">
+				<xsd:annotation>
+					<xsd:documentation>The staircase has regular steps, all of the same height and depth</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="uneven">
+				<xsd:annotation>
+					<xsd:documentation>The steps are not all the same size</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="rough">
+				<xsd:annotation>
+					<xsd:documentation>The steps are very different in size, some may be missing or steeply sloped</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+		</xsd:restriction>
+	</xsd:simpleType>
+	<!-- ======================================================================= -->
+	<xsd:simpleType name="StairRampEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for the Stair Ramp</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="none"/>
+			<xsd:enumeration value="bicycle">
+				<xsd:annotation>
+					<xsd:documentation>A narrow ramp on the side, designed to push a bicycle</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="luggage">
+				<xsd:annotation>
+					<xsd:documentation>A ramp on the side, designed to roll a suitcase</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="stroller">
+				<xsd:annotation>
+					<xsd:documentation>A pair of ramps with steps in between, intended for pushing a stroller</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="other">
+				<xsd:annotation>
+					<xsd:documentation>Another type of handrail included in the staircase itself</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+		</xsd:restriction>
+	</xsd:simpleType>
+	<!-- ======================================================================= -->
+	<xsd:simpleType name="ReachedFloorAnnouncementEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for the Reached Floor Announcement</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="none"/>
+			<xsd:enumeration value="visual"/>
+			<xsd:enumeration value="tactile"/>
+			<xsd:enumeration value="audio"/>
+			<xsd:enumeration value="visualAndAudio"/>
+			<xsd:enumeration value="visualAndAudioAndTactile"/>
+		</xsd:restriction>
+	</xsd:simpleType>
+	<!-- ======================================================================= -->
+	<xsd:simpleType name="OtherDoorsEnumeration">
```

Comment:
Shouldn't this be considered together with #535 before deciding to add the enums - or is the preferred way to first close this PR and then possibly undo things again? 

@Aurige  and @Joostb61 mentioned the simpler alternative of using notices and ThroughLoader (which wouldn't require any change).

@Joostb61  mentioned an alternative of compass octets. @duexw : When used in a LiftCallEquipment - would that be of value for indoor routing? One would know from which side to approach an EquipmentPlace that contains a lift.